LICENSE.header.txt

```diff
@@ -0,0 +1,4 @@
+Copyright (c) 2021, salesforce.com, inc.
```

Comment:
Why `LICENSE.header.txt` and not just `LICENSE.txt ` as recommended here: https://github.com/salesforce/oss-template/blob/master/license_info.md ?

Reply:
I see other open source repos using just `LICENSE` + VS Code uses a special 🔑  icon next to license.

![license-icon](https://user-images.githubusercontent.com/46537641/143064611-ea3ecca3-6e02-480e-a5f5-34a377de3d9c.png)

It seems it's ok to use `LICENSE`, `LICENSE.txt`, or `LICENSE.md`, but it's most common to have `LICENSE.txt`.  I renamed the license file to `LICENSE.txt` to comply.

![licnse txt-icon](https://user-images.githubusercontent.com/46537641/143064763-ef5283db-414e-4942-b8b3-03ee977516fd.png)